starboard/nplb/compiler_compliance/cpp20_support.cc

```diff
@@ -0,0 +1,75 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the aLicense is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <iostream>
+#include <memory>
+#include <numeric>
+#include <string>
+#include <string_view>
+#include <utility>
+#include <vector>
+
+namespace starboard {
+namespace nplb {
+namespace compiler_compliance {
+namespace {
+
+// These examples are taken after referring to
+// 1) The chromium C++20 allowlist:
```

Comment:
I recommend amending the comment so say "C++20 allowlist from m114 milestone branch". 

The `branch-heads/5735` would be really obscure to unsuspecting reader otherwise.

Reply:
done, good point.